src/components/molecules/ZopaFooter/__snapshots__/ZopaFooter.test.tsx.snap

```diff
@@ -597,15 +597,15 @@ and we are registered with the Office of the Information Commissioner (No. Z8797
           >
             <svg
               fill="currentColor"
-              height="30px"
+              height="24px"
```

Comment:
is the size change intended here?

Reply:
Good catch 🧐 💯 

I see [on `<SocialLinks />`](https://github.com/zopaUK/react-components/blob/master/src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx#L33) we were explicitly setting their width at `30px` however their default one was set to `24xpx` 🤷🏻‍♂️

I'll update so that both have a with of `30px` instead. 👍🏻